src/content/en/fundamentals/getting-started/primers/async-functions.md

```diff
@@ -0,0 +1,317 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Async functions allow you to write promise-based code as if it were synchronous
+
+{# wf_published_on: 2016-10-07 #}
+{# wf_updated_on: 2016-10-07 #}
+
+# Async functions - making promises friendly {: .page-title }
+
+{% include "web/_shared/contributors/jakearchibald.html" %}
+
+Async functions are enabled by default in Chrome 55 and they're quite frankly
+marvelous. They allow you to write promise-based code as if it were synchronous,
+but without blocking the main thread. They make your asynchronous code less
+"clever" and more readable.
+
+Async functions look like this:
+
+    async function myFirstAsyncFunction() {
+      try {
+        const fulfilledValue = await promise;
+      }
+      catch (rejectedValue) {
+        // …
+      }
+    }
+
+If you use the `async` keyword before a function definition, you can then use
```

Comment:
I guess the reasons are similar to why `function()*` is necessary for generators, but I'm not 100% sure.

FWIW I've been caught out by generators in Python in the past. It wasn't clear to me that a function would behave very differently because it has a `yield` hidden away in it somewhere.